moto/rds/models.py

```diff
@@ -2669,9 +2669,15 @@ def restore_db_cluster_to_point_in_time(
         return self.create_db_cluster(new_cluster_props)
 
     def failover_db_cluster(
-        self, db_cluster_identifier: str, target_db_instance_identifier: str
+        self, db_cluster_identifier: str, target_db_instance_identifier: str = ""
     ) -> DBCluster:
         cluster = self.clusters[db_cluster_identifier]
+
+        if not target_db_instance_identifier:
+            for instance_id, instance in self.databases.items():
```

Comment:
You don't want to search through all databases, do you?  Don't you just want `cluster.members`?